Rancher PAYG offerings for AWS and Azure (#1008)
rancher/rancher-docs: docs/integrations-in-rancher/cloud-marketplace/aws-marketplace-payg-integration/rancher-prime-aws.md

---
title: Prerequisites
---

Before using Rancher Prime on AWS as a pay-as-you-go (PAYG) offering, you need the following resources, information, and tools:

- A Rancher-compatible EKS cluster. Please see the [Rancher support matrix](https://www.suse.com/suse-rancher/support-matrix/all-supported-versions/) for more details. Please refer to [Creating an EKS cluster](https://ranchermanager.docs.rancher.com/getting-started/installation-and-upgrade/install-upgrade-on-a-kubernetes-cluster/rancher-on-amazon-eks#creating-an-eks-cluster-for-the-rancher-server) for bringing up an EKS cluster to [install Rancher Prime PAYG](installing-rancher-prime.md).

> - An ingress installed on the EKS cluster so that Rancher is accessible outside of the cluster. Please refer to [Rancher documentation](https://ranchermanager.docs.rancher.com/getting-started/installation-and-upgrade/install-upgrade-on-a-kubernetes-cluster/rancher-on-amazon-eks#5-install-an-ingress) for instructions on how to deploy Ingress-INGINX on an EKS cluster.

Reviewer: ```suggestion
- An ingress installed on the EKS cluster so that Rancher is accessible outside of the cluster. Please refer to [Rancher documentation](https://ranchermanager.docs.rancher.com/getting-started/installation-and-upgrade/install-upgrade-on-a-kubernetes-cluster/rancher-on-amazon-eks#5-install-an-ingress) for instructions on how to deploy Ingress-NGINX on an EKS cluster.
```

> - Get the Load Balancer IP. Please refer to [Rancher documentation](https://ranchermanager.docs.rancher.com/getting-started/installation-and-upgrade/install-upgrade-on-a-kubernetes-cluster/rancher-on-amazon-eks#6-get-load-balancer-ip) and Save the EXTERNAL-IP.

Reviewer: ```suggestion
- Get the Load Balancer IP. Please refer to [Rancher documentation](https://ranchermanager.docs.rancher.com/getting-started/installation-and-upgrade/install-upgrade-on-a-kubernetes-cluster/rancher-on-amazon-eks#6-get-load-balancer-ip) and save the EXTERNAL-IP.
```

> ---

Reviewer 2: I started leaving comments, but there are a lot of line breaks in the middle of sentences in this file, which makes it hard to make edit suggestions. Could you clean up the line breaks?

title: Rancher Prime in AWS
---

## Prerequisites

> Before using Rancher Prime on AWS as a PAYG offering, you need the following resources, information, and tools:

Reviewer 2: ```suggestion
Before using Rancher Prime on AWS as a pay-as-you-go (PAYG) offering, you need the following resources, information, and tools:
```

> - A Rancher-compatible EKS cluster. Please see
>    [Rancher support matrix](https://www.suse.com/suse-rancher/support-matrix/all-supported-versions/)
>    for more details.

Reviewer 2: ```suggestion
- A Rancher-compatible EKS cluster. Please see the [Rancher support matrix](https://www.suse.com/suse-rancher/support-matrix/all-supported-versions/) to see which versions are supported.
```
